test/plugins/individual-collateral/origin/OETHCollateral.test.ts

```diff
@@ -0,0 +1,259 @@
+}
+
+const increaseTargetPerRef = async (
+  ctx: WOETHCollateralFixtureContext,
+  pctIncrease: BigNumberish
+) => {
+  const lastRound = await ctx.chainlinkFeed.latestRoundData()
+  const nextAnswer = lastRound.answer.add(lastRound.answer.mul(pctIncrease).div(100))
+  await ctx.chainlinkFeed.updateAnswer(nextAnswer)
+}
+
+const reduceRefPerTok = async (ctx: WOETHCollateralFixtureContext, pctDecrease: BigNumberish) => {
+  const slot = 2
+  const storedTotalSupply = BigNumber.from(await getStorageAt(ctx.tok.address, slot))
+  const newStoredTotalAssets = storedTotalSupply.add(storedTotalSupply.mul(pctDecrease).div(100))
+  await setStorageAt(ctx.tok.address, slot, newStoredTotalAssets)
+}
+
+const increaseRefPerTok = async (ctx: WOETHCollateralFixtureContext, pctIncrease: BigNumberish) => {
+  const slot = 2
+  const storedTotalSupply = BigNumber.from(await getStorageAt(ctx.tok.address, slot))
+  const newStoredTotalAssets = storedTotalSupply.sub(storedTotalSupply.mul(pctIncrease).div(100))
+  await setStorageAt(ctx.tok.address, slot, newStoredTotalAssets)
+}
+
+const getExpectedPrice = async (ctx: WOETHCollateralFixtureContext): Promise<BigNumber> => {
+  const uoaPerTargetChainlinkFeedAnswer = await ctx.uoaPerTargetChainlinkFeed.latestAnswer()
+  const uoaPerTargetChainlinkFeedDecimals = await ctx.uoaPerTargetChainlinkFeed.decimals()
+  const targetPerRefChainlinkFeedAnswer = await ctx.chainlinkFeed.latestAnswer()
+  const targetPerRefChainlinkFeedDecimals = await ctx.chainlinkFeed.decimals()
+
+  const refPerTok = await ctx.collateral.refPerTok()
+
+  const result = uoaPerTargetChainlinkFeedAnswer
+    .mul(targetPerRefChainlinkFeedAnswer)
+    .mul(refPerTok)
+    .div(bn(10).pow(uoaPerTargetChainlinkFeedDecimals + targetPerRefChainlinkFeedDecimals))
+
+  return result
+}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificConstructorTests = () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const collateralSpecificStatusTests = () => {}
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const beforeEachRewardsTest = async () => {}
+
+/*
+  Run the test suite
+*/
+
+const opts = {
+  deployCollateral,
+  collateralSpecificConstructorTests,
+  collateralSpecificStatusTests,
+  beforeEachRewardsTest,
+  makeCollateralFixtureContext,
+  mintCollateralTo,
+  reduceTargetPerRef,
+  increaseTargetPerRef,
+  reduceRefPerTok,
+  increaseRefPerTok,
+  getExpectedPrice,
+  itClaimsRewards: it.skip,
+  itChecksTargetPerRefDefault: it,
+  itChecksTargetPerRefDefaultUp: it,
+  itChecksRefPerTokDefault: it,
+  itChecksPriceChanges: it,
+  itChecksNonZeroDefaultThreshold: it.skip,
+  itHasRevenueHiding: it,
+  resetFork: getResetFork(FORK_BLOCK),
+  collateralName: 'OETHCollateral',
+  chainlinkDefaultAnswer: defaultAnswers.targetPerRefChainlinkFeed,
+  itIsPricedByPeg: true,
+  itHasOracleRefPerTok: false,
+  targetNetwork: 'mainnet',
+  toleranceDivisor: bn('1e2'),
```

Comment:
can we increase this toleranceDivisor? most other collateral use 1e8 or 1e9. The only case we have of 1e2 is aerodrome plugins or LP tokens collateral because those are harder to measure, but we don't have that going on here

Reply:
@clement-ux tagging you as this is a new branch/pr

Reply:
Seems like `1e3` is the highest we can go. Should we dig deeper? ITs in the "increase" price functions that the comparison fails maybe need to be adjusted.

Reply:
@julianmrodri Sorry I think I didn't get it. 
Are you waiting for me to modify `toleranceDivisor`? Or this is a conversation you have internally? 
Because I don't see any failing test on the CI.

Reply:
Let me check with Taylor, but it is true we should probably be able to increase it in the test. But if we do that the test fails which indicates to me we can improve the test somewhere. I havent checked in detail but price comparison tests are impacted so probably the expected price we calculate is off or we are missing updating one of the oracles or something in those lines. If you go to `1e4` you will see failures